docs/concepts/ipfs-gateway.md

```diff
@@ -211,4 +211,4 @@ No. The ipfs.io gateway is one of many portals used to view content stored by th
 
 - [A Practical Explainer for IPFS Gateways – Part 1](https://blog.ipfs.tech/2022-06-09-practical-explainer-ipfs-gateways-1/), [Part 2](https://blog.ipfs.tech/2022-06-30-practical-explainer-ipfs-gateways-2/)
 - [Kubo: Gateway configuration options](https://github.com/ipfs/kubo/blob/master/docs/config.md#gateway)
-- [IPFS HTTP Gateways specification](https://specs.ipfs.tech/http-gateways/) 
\ No newline at end of file
+- [IPFS HTTP Gateways specification](https://specs.ipfs.tech/http-gateways/) 
```

Comment:
I have no idea why there is a change with identical content before and after. But I've tried both editing with vim locally and editing directly through GitHub (all without touching this line at all) and this change occurs in both case.

My guess is some sort of CRLF mismatch that git tried to correct by itself?